### DISCUSSION TITLE
the generated images are mostly pure black

### DISCUSSION THREAD

**Discussion by movelikeriver** [Acly/krita-ai-diffusion#1078] [Q&A] [answered]
tried all sorts of models and modes ("generate", "live", etc)
<img width="522" alt="image" src="https://github.com/user-attachments/assets/16fe65ac-0ae6-44a0-abe5-d25a5ddbc05e">

this doesn't seem to be related to sensitive content at all

OS: apple mac, Sonoma 14.4, M3 pro, 36G ram

**Comment by Acly** [comment, marked as answer]
It's somehow related to Mac (after some system update), but unclear. See #783
